test/connect.test.ts

```diff
@@ -0,0 +1,136 @@
+import Ipfs from 'ipfs-core'
+import { IpfsDaemon } from '@ceramicnetwork/ipfs-daemon'
+import { IpfsTopology } from '@ceramicnetwork/ipfs-topology'
+import { DiagnosticsLogger } from '@ceramicnetwork/common'
+
+import peerlistDevUnstable from '../dev-unstable.json'
+import peerlistTestnetClay from '../testnet-clay.json'
+import peerlistMainnet from '../mainnet.json'
+
+const ceramicNetwork: string = process.env.JEST_CERAMIC_NETWORK || ''
+
+const peerlistByNetwork: { [key: string]: string[] } = {
```

Comment:
```suggestion
const peerlistByNetwork: Record<string, string[]> = {
```

Reply:
Why is this better @ukstv ?

Reply:
More concise, idiomatic. Less brackets to comprehend :) Functionally the same.